### ISSUE TITLE
Module to integrate with Eventbrite

### ISSUE THREAD

**Issue by willgearty** [learning-unlimited/ESP-Website#2568]
Sadly, Stanford can't use the credit card module and needs to use Eventbrite to collect credit card payments instead. It would be nice to integrate this with the website (right now we just have a widget on a template-overridden /extraform page). It looks like Eventbrite has an [API](https://www.eventbrite.com/developer/v3/), which is good, but it doesn't look like you can actually process payments through the API. Payments could be made through a widget of some sort, and then we could receive notification of the successful payments via [webhooks](https://www.eventbrite.com/developer/v3/api_overview/webhooks/) and then somehow associate these payments with particular accounts?

For the moment, I'm just putting this here to remind me to look into this possibility at some point in the not-too-distant future (unless this would also help other programs).

**Comment by benjaminjkraft** [comment]
Yeah, probably we would link users to the eventbrite page, and use the webhook to know that they paid.

**Comment by willgearty** [comment]
Resources for later:

[Eventbrite Python SDK](https://github.com/eventbrite/eventbrite-sdk-python)
[Eventbrite Webhooks Tutorial](https://www.eventbrite.com/developer/v3/resources/webhooks_tutorial/) (and [code](https://github.com/eventbrite/webhooks-tutorial-flask))
[Eventbrite Webhooks](https://www.eventbrite.com/developer/v3/api_overview/webhooks/)

**Comment by willgearty** [comment]
#2623 is a sufficient (and more flexible) fix for this at the moment, but I'll leave this open in case someone (incl. me) wants to work on something with webhooks at a later time.
